### PR TITLE
Adding support for older testng versions

### LINKED ISSUE #1540: New version of test runner doesn't work with older versions of testng
The `getParameterTypes` method is not available in testng before version 7.5 and as such the extension cannot run testng tests older than version 7.5 until the usage of this method is replaced with a suitable alternative.

### PR DESCRIPTION
**Pull request by Kropie**
This will fix #1540

### CODE REVIEW

File: test/test-projects/old-testng/pom.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <artifactId>test</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <name>test</name>
+  <url>http://maven.apache.org</url>
+  <dependencies>
+    <dependency>
+      <groupId>org.testng</groupId>
+      <artifactId>testng</artifactId>
+      <version>6.10</version>
```
Comment (jdneo) [inline test/test-projects/old-testng/pom.xml:14]: Let's use `6.9.13.3`, since that's the lowest version we declare to support.

**Comment by jdneo** [comment]
I've update the testng version in the test project to `6.9.13.3`.

The PR looks good to me. I'll merge it and arrange a hotfix release.

I'll check if it is possible to do some automation tests towards different versions of the test framework. Let me know if you have any good idea about that :)

File: test/test-projects/old-testng/pom.xml (hunk 1 above)
Comment (jdneo) [reply test/test-projects/old-testng/pom.xml]: Done

**Review by jdneo — APPROVED** [review]

**Comment by Kropie** [comment]
> I've update the testng version in the test project to `6.9.13.3`.
> 
> 
> 
> The PR looks good to me. I'll merge it and arrange a hotfix release.
> 
> 
> 
> I'll check if it is possible to do some automation tests towards different versions of the test framework. Let me know if you have any good idea about that :)

You could do some simple e2e tests to check that our tests in the various test projects work as expected.

**Comment by jdneo** [comment]
> I am busy for the next few days but I could try to come up with something next week.

It's totally fine. Thank you for all your contributions :)

**Pull request merged**